CI Lint failure in cmd/provider/main.go

The CI workflow failed in the `lint` job.
Run ID: [22483731798](https://github.com/footprint-it-solutions/provider-zonehero/actions/runs/22483731798)
Error:
```
##[error]cmd/provider/main.go:59:12: Error return value of `os.Setenv` is not checked (errcheck)
           os.Setenv("HOME", "/tmp")
                    ^
```
Triggered by: @santhakali

CC: @santhakali

--- a/cmd/provider/main.go
+++ b/cmd/provider/main.go
@@ -56,7 +56,10 @@ func main() {
 	// If HOME is unset or set to root (which is common in containers), set it to /tmp
 	// This ensures that the HLB Terraform provider can write credentials to ~/.hlb
 	if home := os.Getenv("HOME"); home == "" || home == "/" {
-		os.Setenv("HOME", "/tmp")
+		if err := os.Setenv("HOME", "/tmp"); err != nil {
+			fmt.Fprintf(os.Stderr, "Cannot set HOME environment variable: %v\n", err)
+			os.Exit(1)
+		}
 	}
 
 	var (

